templates/livepatch/index.html

```diff
@@ -47,10 +47,10 @@ <h2 class="p-heading--four">Apply critical kernel patches without rebooting.</h2
     <div class="col-12">
       <h2>Free for personal use</h2>
       <p>All you need is an Ubuntu One account. Free for 3 machines.</p>
-      <p><a class="p-button--positive"  href="https://auth.livepatch.canonical.com/"><span class="p-link--external">Get Livepatch</span></a></p>
+      <p><a class="p-button--positive"  href="/advantage#livepatch">Get Livepatch</a></p>
```

Comment:
This section is about the free subscription, but the link goes into the table describing paid subscriptions.

Correcting just the link destination would result in two nearby buttons going to exactly the same place. Instead, I suggest merging the two sections: “Livepatch is part of Ubuntu Advantage for Infrastructure, available for physical servers, virtual machines, containers, and desktops. And it’s free for personal use.”

(Alternatively, if someone has asked you specifically for separate sections, they should be further away from each other.)